package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@enterprise-cmcs/mdct-core",
   "license": "CC0-1.0",
-  "version": "0.1.3",
+  "version": "0.0.0",
```

Comment:
I changed this to 0.0.0 in hopes to convey that this is not an accurate reflection of reality. As noted in the PR notes a follow on story will be created to see what can be done about this but the initial attempt was getting ugly and not successful.